src/modules/users/users.conf

```diff
@@ -145,47 +144,27 @@ doAutologin:     true
 # To require specific password characteristics:
 #  - set 'allowWeakPasswords' to false (the default)
 #  - configure password-checking, e.g. with NIST settings
+#
+# The defaults follow NIST-800-63B guidelines.
 
-
-# These are very weak -- actually, none at all -- requirements
 passwordRequirements:
-    minLength: -1  # Password at least this many characters
-    maxLength: -1  # Password at most this many characters
+    minLength: 8
+    maxLength: 64
     libpwquality:
-        - minlen=0
-        - minclass=0
-
-# These are "you must have a password, any password" -- requirements
-#
-# passwordRequirements:
-#     minLength: 1
-
-# These are requirements the try to follow the suggestions from
-#     https://pages.nist.gov/800-63-3/sp800-63b.html , "Digital Identity Guidelines".
-# Note that requiring long and complex passwords has its own cost,
-# because the user has to come up with one at install time.
```

Comment:
We can lose this editorializing, but then we need to have documentation explaining the choices distro's can make, e.g. "the uncommented settings ask for a strong password and disable autologin, but the user can explicitly change that to enable weak(er) passwords and/or autologin." Again, with examples.